build.gradle

```diff
@@ -11,27 +10,55 @@ repositories {
 }
 
 group = "com.github.hiroyuki_sato"
-version = "0.3.2"
+version = "0.4.0-SNAPSHOT"
 description = "Parses JSON files with JSONPath read by other file input plugins."
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-    compileOnly "org.embulk:embulk-core:0.9.23"
+    compileOnly "org.embulk:embulk-api:0.10.29"
+    compileOnly "org.embulk:embulk-spi:0.10.29"
+
+    compile("org.embulk:embulk-util-config:0.2.+") {
```

Comment:
Not forcing, but I basically don't recommend to use this style of version declaration: `0.2.+`. It's better for (Embulk plugin) developers to be explicitly aware of what versions of libraries used. (`.lockfile` shows it, though.)